src/libs/antares/study/area/scratchpad.cpp

```diff
@@ -57,8 +57,8 @@ void CalculateDailyMeanPower(const Matrix<double>::ColumnType& hourlyColumn,
 }
 
 AreaScratchpad::AreaScratchpad(const StudyRuntimeInfos& rinfos, Area& area) :
- meanMaxDailyGenPower(area.hydro.series->timeseriesNumbersHydroMaxPower),
- meanMaxDailyPumpPower(area.hydro.series->timeseriesNumbersHydroMaxPower)
+  meanMaxDailyGenPower(area.hydro.series->timeseriesNumbersHydroMaxPower, "dailyGenPower"),
```

Comment:
As recommended in [PR 2073](https://github.com/AntaresSimulatorTeam/Antares_Simulator/pull/2073) (please see [this comment](https://github.com/AntaresSimulatorTeam/Antares_Simulator/pull/2073#discussion_r1591156960)), we should not have to make this change